packages/logging/src/environment.ts

```
@@ -0,0 +1,76 @@
+ * Detect current environment from NODE_ENV
+ */
+export function detectEnvironment(): Environment {
+  const nodeEnv = process.env.NODE_ENV?.toLowerCase()
+
+  switch (nodeEnv) {
+    case 'test':
+      return 'test'
+    case 'production':
+    case 'prod':
+      return 'production'
+    case 'development':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)